master/buildbot/process/properties.py

```diff
@@ -603,6 +613,17 @@ def _parse_src(arg):
             codebase = attr = repl = None
         return _SourceStampDict(codebase), attr, repl
 
+    def _parse_worker(self, arg):
+        try:
+            prop, repl = arg.split(":", 1)
+        except ValueError:
+            prop, repl = arg, None
+        if not Interpolate.identifier_re.match(prop):
```

Comment:
not sure if we really need that check. If somebody decides to name an info file with chineese characters, this would not be a techical problem, but this check will make it unavailable..

Reply:
why is the check done for properties and keywords?